circle-sensor-card.js

```diff
@@ -113,18 +113,34 @@ class CircleSensorCard extends LitElement {
 
   _updateConfig() {
     const container = this._root.querySelector('.labelContainer');
-    container.style.color = this.config.font_color || 'var(--primary-text-color)';
-    container.style.fontSize = this.config.font_size || '1em';
+    container.style.color = 'var(--primary-text-color)';
+
+    if (this.config.font_style) {
+      Object.keys(this.config.font_style).forEach((prop) => {
+        container.style.setProperty(prop, this.config.font_style[prop]);
+      });
+    }
   }
 
   set hass(hass) {
     this.state = hass.states[this.config.entity];
-    if (!this.state || isNaN(this.state.state)) {
-      console.error(`State is not a number: ${this.state.state}`);
-      return;
+    
+    if (this.config.attribute) {
+      if (!this.state.attributes[this.config.attribute] || 
+          isNaN(this.state.attributes[this.config.attribute].replace(/,/g,""))) {
```

Comment:
I don't think this is a good idea since different locales use a comma to mean different things and I don't know if we really want to get into l10n with this. Are there platforms that return numerical values as strings with thousands separators?

Reply:
Yes, the fitbit sensor, https://www.home-assistant.io/components/sensor.fitbit/, which was my main reason for submitting this PR. Are you suggesting that it would be more prudent to update the sensor? I can get behind that.

Reply:
Yes, I think that is the right way to handle this. If it is returning a value that is meant to be numeric, it should actually be numeric and not a string representation with locale specific symbols

Reply:
Sounds good. I'll make some template_sensors for now and work on a PR for fitbit component ASAP and remove this for now.

Reply:
Sounds good